returnn/training.py

```diff
@@ -376,6 +367,16 @@ def create_returnn_config(
         if returnn_config.post_config is not None:
             post_config.update(copy.deepcopy(returnn_config.post_config))
 
+        if keep_epochs:
+            if isinstance(post_config["cleanup_old_models"], bool):
+                post_config["cleanup_old_models"] = {"keep": keep_epochs}
+            elif isinstance(post_config["cleanup_old_models"], dict):
+                post_config["cleanup_old_models"]["keep"] = keep_epochs
```

Comment:
Would we maybe check if already some keep_epochs are defined here and if yes (use union? give warning? use defined value? overwrite anyway?)

Reply:
Ah right, this is missing. I would say an Exception is the best solution...